app/docker/views/containers/logs/containerLogsController.js

```diff
@@ -30,7 +31,7 @@ function ($scope, $transition$, $interval, ContainerService, Notifications, Http
   function setUpdateRepeater(skipHeaders) {
     var refreshRate = $scope.state.refreshRate;
     $scope.repeater = $interval(function() {
-      ContainerService.logs($transition$.params().id, 1, 1, $scope.state.displayTimestamps ? 1 : 0, $scope.state.lineCount, skipHeaders)
+      ContainerService.logs($transition$.params().id, 1, 1, $scope.state.displayTimestamps ? 1 : 0, strToTimestamp($scope.state.sinceTimestamp), $scope.state.lineCount, skipHeaders)
```

Comment:
You can remove the usage of the `strToTimestamp` function and replace it with something like:

```suggestion
      ContainerService.logs($transition$.params().id, 1, 1, $scope.state.displayTimestamps ? 1 : 0, moment($scope.state.sinceTimestamp, 'X'), $scope.state.lineCount, skipHeaders)
```

See moment docs: https://momentjs.com/docs/#/parsing/string-format/